interview_db/management/commands/create_sample_data.py

```diff
@@ -0,0 +1,290 @@
+""" Management command to django's manage.py called
+    create_sample_data that will generate interviews for testing.
+"""
+from django.core.management.base import BaseCommand
+from interview_db.models import *
+from decimal import *
+
+
+class Command(BaseCommand):
+    help = "Creates new data for testing"
+
```

Comment:
Like with spotseeker's create_sample_spots, you should probably ask for confirmation here, wouldn't want someone to accidentally erase all the production data by accidentally running this! Alternatively, I guess you could check for ENV="localdev" and only allow running it in that case.